src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ParameterExpander.java

```diff
@@ -249,6 +293,61 @@ private Integer getBuildStartedCodeReviewValue(Run r) {
         return map;
     }
 
+    /**
```

Comment:
I don't see a reason to copy-paste javadoc.
See [StackOverflow](https://stackoverflow.com/questions/2558695/javadoc-reuse-for-and-overloaded-methods)

Reply:
Removed